Author: Tigro <ashejn@gmail.com>

Fix mk-tpl-config.sh to not fail when guile headers don't use noreturn

https://abf.rosalinux.ru/import/autogen/commit/d773f874326700a8aeef935351dab2b915575a7c

diff -up autogen-5.18.3/autoopts/mk-tpl-config.sh.mktplconfig autogen-5.18.3/autoopts/mk-tpl-config.sh
--- autogen-5.18.3/autoopts/mk-tpl-config.sh.mktplconfig	2014-05-10 17:48:26.000000000 +0200
+++ autogen-5.18.3/autoopts/mk-tpl-config.sh	2014-05-27 14:05:26.961175451 +0200
@@ -200,9 +200,8 @@ fix_guile() {
     cd ${builddir}
     find_libguiledir "${LGCFLAGS}"
 
-    list=`set +e ; exec 2>/dev/null
-        find ${libguiledir}/libguile* -type f | \
-            xargs grep -l -E '\<noreturn\>'`
+    list=`find ${libguiledir}/libguile* -type f | \
+            xargs grep -l -E '\<noreturn\>' 2> /dev/null || :`
 
     test -z "$list" && exit 0
 
